.github/workflows/nightly.yml

```diff
@@ -1,9 +1,12 @@
 ---
 name: "Release"
 on:
-  schedule:
-#    - cron: '10 0 * * *'
-    - cron: '20 17 * * *'
+  push:
+    branches:
+      - 'add-docker-build-variants'
```

Comment:
@iogakos what is add-docker-build-variants branch?

Reply:
This is part of [81a7f5f](https://github.com/Metrika-Inc/agent/pull/81/commits/81a7f5ff939a2b644d90e8f72ae2d9b756f80673) which I've been using to test the CI while changing the branch. Will revert upon approval.

Reply:
Ok I see, thanks